### DISCUSSION TITLE
exam inspection

### DISCUSSION THREAD

**Discussion by DezignDezire** [sebastian-hofstaetter/teaching#113] [General]
Hey, will there be an exam inspection?
LG

**Comment by sophiaalthammer** [comment]
Hey, 

yes there will be an exam inspection in TUWEL, where you can see the test and the answers. It will be together with the second exam after the second exam is graded as well and will be announced via TUWEL announcements.

Best,
Sophia

**Reply by abcitat** [reply]
Hi!

is there any update about the exam inspection?

Best,
TANG Fung Yee
